packages/web-components/@sfx/base/test/base.test.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, spy, stub } from './utils';
+import Base from '../src/base';
+import { LitElement } from 'lit-element';
+import { TemplateResult } from 'lit-element';
+import * as BaseUtils from '../src/utils';
+
+
+describe('Base Class', () => {
+  let base: any = {}
+  beforeEach(() => {
+    base = new Base();
+  });
+
+  it('should extend from LitElement', () => {
+    const base = new Base();
```

Comment:
This line is not necessary because the `base` in the outer scope already gives you what you want.